src/app.js

```diff
@@ -4,6 +4,7 @@ import sslRedirect from 'heroku-ssl-redirect';
 import nakedRedirect from 'express-naked-redirect';
 import express from 'express';
 import compression from 'compression';
+import {createProxyMiddleware} from "http-proxy-middleware";
```

Comment:
Moving these to the top-level will also necessitate moving them from `devDependencies` to `dependencies` in `package.json`, I think.

Reply:
I updated to keep the imports at their existing levels, so that's no longer necessary: 7cc84e85f8a8c557fe9c96ff787bceae774dd2c5